### DISCUSSION TITLE
Accessing the DSC from the outside

### DISCUSSION THREAD

**Discussion by aitorcelaya** [International-Data-Spaces-Association/DataspaceConnector#80] [Q&A] [answered]
I am trying to access the Dataspace Connector as a provider from another Connector acting as a consumer. How do we request the information without the need of the admin user to access the /api path? 

Do we NEED that admin user for every interaction with the Dataspace Connector? 

https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/Provider shows how to create resources as a provider and offer them to the Broker, but I am unable to find my endpoints as a non-admin.

We are trying to access the DSC in the Integration Test Camp 

Thank you!

**Comment by HeinrichPet** [comment, marked as answer]
Hi,
you can find the IDS message sequence here: [https://github.com/International-Data-Spaces-Association/IDS-G-pre/tree/connector-interaction/Communication/sequence-diagrams/data-connector-to-data-connector#message-flows-for-connector-to-connector-communication](https://github.com/International-Data-Spaces-Association/IDS-G-pre/tree/connector-interaction/Communication/sequence-diagrams/data-connector-to-data-connector#message-flows-for-connector-to-connector-communication )
This is currently only available for IDSA members.

Most of it is also available in the [DSC documentation](https://international-data-spaces-association.github.io/DataspaceConnector/Documentation/Messages)

**Reply by HeinrichPet** [reply]
Oh and the API endpoint over which this communication runs is `/api/ids/data`.
This endpoint is not secured via credentials (but via DAPS token).
